### DISCUSSION TITLE
Is it time to change the licence?

### DISCUSSION THREAD

**Discussion by xenu** [hsutter/cppfront#1084] [Suggestions]
Now that the first release has been made, is it time to change the licence to something more permissive than CC-BY-NC-ND-4.0?

Related discussion from two years ago: https://github.com/hsutter/cppfront/issues/122

**Comment by hsutter** [comment]
Yes, I'm actively working on a license that allows for commercial use. I have no ETA to announce yet, but I'd like to have it in place this calendar year if possible.
